docs/visualization/plotting-overview.ipynb

```diff
@@ -232,38 +274,39 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d['data_2d'].attrs['T'] = sc.Variable(dims=['x'], values=np.linspace(3, 50, M), unit=sc.units.K )\n",
-    "sc.plot(d['data_2d'], labels={'x': 'T'})"
+    "da1.plot(scale={'x': 'log'})"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "To control which dimensions are shown along which axes of the matplotlib figure use the transpose button in the toolbar, or transpose data before calling `plot`, e.g."
+    "da.plot(scale={'x': 'log', 'time': 'log'})"
```

Comment:
**Optional:**

Again, the plot with log scales looks almost the same as the one with linear axes
![Screenshot at 2022-10-14 13-05-06](https://user-images.githubusercontent.com/39047984/195832467-5d1ecaba-0df7-44d6-a953-d5f6f7164e25.png)

We could make 2d data where this is better highlighted?
We could also make a 2d example with `norm='log'`?

Reply:
We could. Is it important here?

Reply:
I am aiming for concise code, since the focus of this page is to explain how to do it. The more setup code we have the less clear it will be.

Reply:
> Is it important here?

Not so much, which is why I wrote "Optional" at the start of my comment ;-)